drivers/i2c/i2c_ll_stm32.c

```diff
@@ -28,18 +46,24 @@ int i2c_stm32_runtime_configure(struct device *dev, u32_t config)
 	u32_t clock = 0;
 
 #if defined(CONFIG_SOC_SERIES_STM32F3X) || defined(CONFIG_SOC_SERIES_STM32F0X)
-	LL_RCC_ClocksTypeDef rcc_clocks;
-
-	/*
-	 * STM32F0/3 I2C's independent clock source supports only
-	 * HSI and SYSCLK, not APB1. We force clock variable to
-	 * SYSCLK frequency.
-	 */
-	LL_RCC_GetSystemClocksFreq(&rcc_clocks);
-	clock = rcc_clocks.SYSCLK_Frequency;
+	if (cfg->ll_clock_source != LL_I2C_NO_CLOCKSOURCE) {
```

Comment:
Remove from this change

Reply:
There were compile errors in CI before !???!  Is this gone?